docs/components/unstyled/sign-out-button.mdx

```diff
@@ -0,0 +1,160 @@
+import { Tables } from "@/components/Table";
+import { CodeBlockTabs } from "@/components/CodeBlockTabs";
+
+# `<SignOutButton />`
+
+The `<SignOutButton/>` component is a button that signs a user out. By default, it is a `<button>` tag that says **Sign Out**, but it is completely customizable by passing children.
+
+## Usage
+
+#### Basic Usage
+
+<CodeBlockTabs options={['Next.js', 'React', 'Remix', 'Gatsby']}>
+  ```jsx copy filename="pages/index.js"
+  import { SignOutButton } from "@clerk/nextjs";
+
+  export default function Home() {
+    return (
+      <div>
+      <h1> Sign out </h1>
+        <SignOutButton />
+      </div>
+    );
+  }
+  ```
+
+  ```jsx copy filename="example.js"
+  import { SignOutButton } from "@clerk/clerk-react";
+
+  export default function Example() {
+    return (
+      <div>
+      <h1> Sign out </h1>
+        <SignOutButton />
+      </div>
+    );
+  }
+  ```
+
+  ```jsx copy filename="pages/index.js"
+  import { SignOutButton } from "@clerk/remix";
+
+  export default function Home() {
+    return (
+      <div>
+      <h1> Sign out </h1>
+        <SignOutButton />
+      </div>
+    );
+  }
+  ```
+
+  ```jsx copy filename="pages/index.js"
+  import { SignOutButton } from "gatsby-plugin-clerk";
+
+  export default function Home() {
+    return (
+      <div>
+      <h1> Sign out </h1>
+        <SignOutButton />
+      </div>
+    );
+  }
+  ```
+</CodeBlockTabs>
+
+#### Custom Usage
+
+In some cases you will want to use your own button, or button text. You can do that by wrapping your button up.
+
+<CodeBlockTabs options={['Next.js', 'React', 'Remix', 'Gatsby']}>
+  ```jsx copy filename="pages/index.js"
+  import { SignOutButton } from "@clerk/nextjs";
+
+  export default function Home() {
+    return (
+      <div>
+      <h1> Sign out </h1>
+        <SignOutButton>
+        <button>Sign in with Clerk</button>
+        </SignOutButton>
+      </div>
+    );
+  }
+  ```
+
+  ```jsx copy filename="example.js"
+  import { SignOutButton } from "@clerk/clerk-react";
+
+  export default function Example() {
+    return (
+      <div>
+      <h1> Sign out </h1>
+        <SignOutButton>
+        <button>Sign in with Clerk</button>
+        </SignOutButton>
+      </div>
+    );
+  }
+  ```
+
+  ```jsx copy filename="pages/index.js"
+  import { SignOutButton } from "@clerk/remix";
+
+  export default function Home() {
+    return (
+      <div>
+      <h1> Sign in </h1>
+        <SignOutButton>
+        <button>Sign in with Clerk</button>
+        </SignOutButton>
+      </div>
+    );
+  }
+  ```
+
+  ```jsx copy filename="pages/index.js"
+  import { SignOutButton } from "gatsby-plugin-clerk";
+
+  export default function Home() {
+    return (
+      <div>
+      <h1> Sign out </h1>
+        <SignOutButton>
+        <button>Sign in with Clerk</button>
+        </SignOutButton>
+      </div>
+    );
+  }
+  ```
+</CodeBlockTabs>
+
+## Props
+
+<Tables
+  headings={["Name", "Type", "Description"]}
+  rows={[
```

Comment:
Eliminates the X scroll on long descriptions like we were seeing. Fixes the issue of description tabs being too cramped like we were seeing. Temporary style fix - I have a note to style tables later.

```suggestion
  headingsMeta={[{maxWidth:'300px'}, {maxWidth:'300px'}, {maxWidth:'300px'}]}
  rows={[
```